generate_upstream_list.py

```diff
@@ -9,27 +9,23 @@
 
 def get_commit_log(git_repo, viewvc_url, author_in_commit_body=False):
     repo = Repo(git_repo)
-    log = repo.commits(config.EMAIL_GREP, search_body=author_in_commit_body)
+    email_grep = '(Contributed|Patch) (from|by).*@opera.com'
```

Comment:
Oh, OK.  I guess it would probably be sufficient to move the domain to the config file?